TreeMaker/python/maker.py

```diff
@@ -101,6 +102,8 @@ def __init__(self,parameters):
         self.VarsInt                    = cms.vstring()
         self.VarsBool                   = cms.vstring()
         self.VectorTLorentzVector       = cms.vstring()
+        self.VectorXYZVector            = cms.vstring()
+        self.VectorXYZPoint             = cms.vstring()

```

Comment:
you should add the other new branch types here also

Reply:
What other ones? vector<vector<object>>?

Reply:
`VarsXYZVector`, etc. Every new line in `treeMaker.py`

Reply:
Fixed